packages/relative-values/src/components/View/ListView/sidebar.tsx

```diff
@@ -45,7 +46,15 @@ value_${numeratorItemName} = dists[0]
 value_${denominatorItemName} = dists[1]
 relativeValue = value_${numeratorItemName} / value_${denominatorItemName}
 median = abs(inv(dists[1], 0.5))
-[Plot.scatter({xDist:dists[1] / median, yDist: dists[0]/median}), relativeValue]`;
+tickFormat = '^10'
```

Comment:
Is `^` here for centering the label? It doesn't seem to play well with my label width detection code in `drawAxes`.
<img width="469" alt="Captura de pantalla 2023-04-24 a la(s) 13 53 24" src="https://user-images.githubusercontent.com/89368/234101618-440292d1-5441-471c-b325-bed48c5bdbaa.png">

Reply:
Ah, I just took it from the examples, didn't realize what it was doing.

Reply:
Tbh I'm not entirely sure why that happened, it's not like d3.format can affect `textAlign` property, it just adds some padding with spaces or maybe invisible space-like symbols. So there might be a bug in my `drawAxes` implementation.

Reply:
Fixed that, that gets around the cutting-off issue.